include/parakeet/ProE/Driver.h

```diff
@@ -117,12 +117,18 @@ class Driver : public mechaspin::parakeet::Driver
         /// \returns The state of the resample filter
         bool isResampleFilterEnabled();
 
-        /// \brief Sets the sensor IPv4 settings
-        /// \param[in] ipAdress - The IP Address the sensor should live on
-        /// \param[in] subnetMask - The subnetMask the sensor should live on
-        /// \param[in] gateway - The gateway the sensor should live on
-        /// \param[in] port - The port that the sensor should be publishing point data from
-        void setIPv4Settings(const unsigned char* ipAddress, const unsigned char* subnetMask, const unsigned char* gateway, const unsigned short port);
+        /// \brief Set the IPv4 settings for the listens on
```

Comment:
"for the listens on"? Seems like you missed some words there. I would suggest:
The IP address and port for the sensor. Messages to the sensor will be sent to this address.

Reply:
Must have undid a part of what I typed at some point. Good catch.